projects/organization-management/src/app/models/b2b-user/b2b-user.model.ts

```diff
@@ -0,0 +1,5 @@
+import { User } from 'ish-core/models/user/user.model';
+
+export interface B2bUser extends User {
+  name?: string; // list call only
```

Comment:
i will do a change request to get firstName, lastName instead. (more compatible)